### ISSUE TITLE
Release: java_tools v11.2

### ISSUE THREAD

**Issue by comius** [bazelbuild/java_tools#45]

**Comment by comius** [comment]
Changes:
```
git log --pretty=format:"%h %an %s" 8cc0e261a3...7a94654 -- src/java_tools/ third_party/ijar/ tools/jdk/BUILD.java_tools
ce7afe730d Ryan Beasley Shade com.google.common.** in the Jacoco test runner.
a3c52a78ea ilist BEGIN_PUBLIC Replace implicit output ":libname.jar" with ":name".
0216ee5441 philwo Fix Bazel #10214: JDK 13 introduced a source compatibility issue.
4fc4868065 ilist Remove support for the Java singlejar implementation.
ba60c0b3f9 Vaidas Pilkauskas ijar: fix manifest sections handling
9a0bcbb012 cushon Header compilation works with VanillaJavaBuilder now
7610755002 Ivo List Fix ijar failing on empty jars.
210ae64e71 dorranh Support path mappings in JacocoLCOVFormatter
299ec8d7b1 Googler Create a new target containing only CurrentRunningTest
```

**Comment by comius** [comment]
Downstream test: https://buildkite.com/bazel/bazel-at-head-plus-downstream/builds/1947
